apps/posts/src/views/Tags/components/TagsList.tsx

```diff
@@ -0,0 +1,145 @@
+                    </TableRow>
+                </TableHeader>
+                <TableBody className="flex flex-col lg:table-row-group">
+                    <SpacerRow height={spaceBefore} />
+                    {visibleItems.map(({virtualItem, item, props}) => {
+                        const shouldRenderPlaceholder =
+                            virtualItem.index > items.length - 1;
+
+                        if (shouldRenderPlaceholder) {
+                            return <PlaceholderRow {...props} />;
+                        }
+
+                        return (
+                            <TableRow
+                                {...props}
+                                className="relative grid w-full grid-cols-[1fr_5rem] items-center gap-x-4 p-2 md:grid-cols-[1fr_auto_5rem] lg:table-row lg:p-0"
+                            >
+                                <TableCell className="static col-start-1 col-end-1 row-start-1 row-end-1 flex min-w-0 flex-col p-0 lg:table-cell lg:w-1/2 lg:p-4 xl:w-3/5">
+                                    <a
+                                        className="block truncate pb-1 text-lg font-medium before:absolute before:inset-0 before:z-10"
+                                        href={`#/tags/${item.slug}`}
+                                    >
+                                        {item.name}
+                                    </a>
+                                    <span className="block truncate text-muted-foreground">
+                                        {item.description}
+                                    </span>
+                                </TableCell>
+                                <TableCell className="col-start-1 col-end-1 row-start-2 row-end-2 flex p-0 lg:table-cell lg:p-4">
+                                    <span className="block truncate">
+                                        {item.slug}
+                                    </span>
+                                </TableCell>
+                                <TableCell className="col-start-1 col-end-1 row-start-3 row-end-3 flex p-0 md:col-start-2 md:col-end-2 md:row-start-1 md:row-end-3 lg:table-cell lg:p-4">
+                                    {item.count?.posts ? (
+                                        <a
+                                            className="relative z-10 -m-4 inline-block p-4 hover:underline"
+                                            href={`#/posts?tag=${item.slug}`}
+                                        >
+                                            {formatNumber(item.count?.posts)}{' '}
+                                            posts
+                                        </a>
+                                    ) : (
+                                        <span className="text-muted-foreground">
+                                            0 posts
+                                        </span>
+                                    )}
```

Comment:
@peterzimon Whats the stance re pluralization? 

The old Ember app used an infliction library. I'd typically rather use a translation library since e.g. message-format has great pluralization support. In the React admin we have nether of those AFAIK.

Reply:
Good point. The Admin is English only ATM and there are no concrete plans for the foreseeable future to translate it, that's why we just went with dumb conditions. Feel free to use a translation library however, that's def. more future proof.

Reply:
In this case it would end up saying `1 posts` which is incorrect, but consistent with the rest of the new admin screens. I can add an inline conditional to fix that if you prefer. 

We could definitely look at adding a translation lib as well. If we want to have translation support, then the rewrite provides a good opportunity to introduce that. Maybe that's something worth discussing in a slightly wider context first tho?

Reply:
From a design POV we should optimize to show correct pluralization, so `1 post` is strongly preferred.